lib/utils/manager/storage_directory_manager.dart

```diff
@@ -13,6 +13,8 @@ class StorageDirectoryManager {
 
   static StorageDirectoryManager get instance => _instance;
 
+  factory StorageDirectoryManager() => _instance;
```

Comment:
why need it?

Reply:
it was for testing purpose but I forgot to remove it when I updated the tests. Fixed now